config/_default/menus/menus.en.toml

```
@@ -0,0 +1,66 @@
+[[main]]
+  name = "Docs"
+  url = "/docs/introduction/overview"
+  weight = 10
+
+[[main]]
+  name = "Reference"
+  weight = 20
+  identifier = "reference"
+
```

```suggestion

[[main]]
  name = "Guides"
    url = "/docs/guides"
  weight = 20
  identifier = "guides"
```

Let's move the guides and tutorials (content in https://libp2p-docs-beta.on.fleek.co/docs/guides/go/) out to a different tab like https://docs.filecoin.io/get-started/overview/